nav2_behavior_tree/plugins/action/clear_costmap_service.cpp

```diff
@@ -32,10 +32,38 @@ void ClearEntireCostmapService::on_tick()
   increment_recovery_count();
 }
 
+ClearExceptRegionService::ClearExceptRegionService(
```

Comment:
https://github.com/ros-planning/navigation2/blob/80bea85151abe264a3f9297694f0f863f7f7dc05/nav2_behavior_tree/test/plugins/action/test_clear_costmap_service.cpp if you added simple test fixtures for these, that would give them test coverage